.kokoro/environment/functions.cfg

```diff
@@ -0,0 +1,19 @@
+# Download resources for environment tests (service account key, etc.)
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/google-cloud-nodejs"
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/nodejs-logging/.kokoro/environment.sh"
+}
+
+# Specify which tests to run
+env_vars: {
+    key: "ENVIRONMENT"
+    value: "functions"
+}
+
+# TODO: actually use this variable somewhere
+env_vars: {
+    key: "RUNTIME"
+    value: "nodejs12"
```

Comment:
I only needed this variable to control different GCF versions, since I needed tests for python37 and python38. You may not need it for yours

Reply:
For sure - I'll leave it in as a TODO for now, since I'll set up these tests on Node14 sometime this year